### PR TITLE
[MIG] stock_picking_purchase_order_link: Migration to 16.0 

### PR DESCRIPTION
**Pull request by hugo-cordoba**
[16.0][MIG] stock_picking_purchase_order_link

### CODE REVIEW

**Review by aliciagaarzo — APPROVED** [review]

File: stock_picking_purchase_order_link/tests/test_stock_picking_purchase_order_link.py
Comment (glitchov) [inline stock_picking_purchase_order_link/tests/test_stock_picking_purchase_order_link.py]: You mean test_picking_to_purchase_order don't you ?

Comment (hugo-cordoba) [reply stock_picking_purchase_order_link/tests/test_stock_picking_purchase_order_link.py]: You are right, I had overlooked the name of the test. I have changed it! Thanks.

**Review by glitchov — APPROVED** [review]

**Comment by glitchov** [comment]
LGTM code + functional

**Comment by hugo-cordoba** [comment]
Good morning @rvalyi  @pedrobaeza, can you check if the migration is correct and merge the development?

**Comment by pedrobaeza** [comment]
/ocabot migration stock_picking_purchase_order_link

At least one PSC should review the PR for merging it:

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#72review

Please review other PRs and ask in exchange that they review yours.

**Comment by hugo-cordoba** [comment]
Thank you for your comment @pedrobaeza.

I have some questions about how to act. How do I know who is PSC? How should I request to review my PR? Should it be after reviewing a PR of yours?

**Review by rousseldenis — APPROVED** [review]
Code  review

**Comment by hugo-cordoba** [comment]
Hi @rousseldenis. 
Thanks for the review and for merge, but something seems to have gone wrong, why?

**Comment by rousseldenis** [comment]
> Hi @rousseldenis. Thanks for the review and for merge, but something seems to have gone wrong, why?

I have relaunched the failed job.

FYI, you can see which one has failed by clicking on Merge PR red cross in this PR (which is yellow now :sweat_smile: ) :

image

**Comment by hugo-cordoba** [comment]
Understood, thank you very much @rousseldenis

**Comment by HugoCordobaLeal** [comment]
@OCA/core-maintainers

**Pull request merged**
